plugin-openshift/che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/OpenshiftServiceClient.java

```diff
@@ -37,6 +37,8 @@
 
     Promise<Project> createProject(ProjectRequest request);
 
+    Promise<Void> deleteProject(String nameSpace);
```

Comment:
deleteProject but parameter named nameSpace. It would be better to name it project